docs/tutorial/security.md

```diff
@@ -709,6 +711,124 @@ const { shell } = require('electron')
 shell.openExternal('https://example.com/index.html')
 ```
 
+## 15) Disable the `remote` module
+
+The `remote` module provides a simple way for the renderer processes to
+access APIsnormally only available in the main process. Using it, a
+renderer can invoke methods of a main process object without explicitly sending
+inter-process messages. If your desktop application does not run untrusted
+content, this is a very elegant way to have your renderer processes access and
+work with modules that are only available to the main process, such as
+GUI-related modules (dialog, menu etc.).
+
+However, if your app can run untrusted content and even if you
+[sandbox][sandbox] your renderer processes accordingly, the `remote` module
+makes it easy for malicious code to escape the sandbox and have access to
+system resources via the higher privileges of the main process. Therefore,
+it should be disabled in such circumstances.
+
+### Why?
+
+`remote` uses an internal IPC channel to communicate with the main process.
+"Prototype pollution" attacks can grant malicious code access to the internal
+IPC channel, which can then be used to escape the sandbox by mimicking `remote`
+IPC messages and getting access to main process modules running with higher
+privileges.
+
+Additionally, if you're not careful, preload scripts may leak modules to a
+sandboxed renderer. Leaking `remote` arms malicious code with a multitude
+of main process modules with which to perform an attack.
+
+Disabling the `remote` module eliminates these attack vectors. Enabling
+context isolation also prevents the "prototype pollution" attacks from
+succeeding.
+
+### How?
+
+Sandboxed renderers
+
+```js
+// Bad
```

Comment:
The bad/good examples make sense here & in the rest of the docs, but I do suspect a lot of people will ignore the text and skip straight down to this.

For that reason, I might modify the comment a little:

```suggestion
// Bad if the renderer can run untrusted content
```